src/Symfony/Component/Cache/Traits/RelayProxy.php

```diff
@@ -456,6 +456,11 @@ public function bitcount($key, $start = 0, $end = -1, $by_bit = false): \Relay\R
         return ($this->lazyObjectState->realInstance ??= ($this->lazyObjectState->initializer)())->bitcount(...\func_get_args());
     }
 
+    public function bitfield($key, ...$args): \Relay\Relay|array|false
```

Comment:
hm.. why was this method added?

Reply:
The test `Symfony\Component\Cache\Tests\Traits\RedisProxiesTest::testRelayProxy` [failed](https://github.com/symfony/symfony/actions/runs/6138815527/job/16656061403#step:8:8501) after update php extension `ext-relay`, just I sync proxy trait with latest changes of `Relay\Relay` class